File: Sorting/In C/BubbleSort.c
//Bubble Sort

#include<stdio.h>

void getArray(int array[],int n){

    for(int i=0;i<n;i++){
        printf("Enter value %d",i+1);
        scanf("%d",&array[i]);
    }
}

void dispArray(int array[],int n){

    for(int i=0;i<n;i++){
        printf("%d ",array[i]);
    }   
}

void BubbleSort(int array[],int n){
    int temp=0;
        for(int i=0;i<n;i++){
            for(int j=1;j<=(n-1-i);j++){
                if(array[j-1]>array[j]){
                    temp=array[j];
                    array[j]=array[j-1];
                    array[j-1]=temp;
                }
            }
            printf("%s",array);
        }
    }

int main(){

    int n,array[50];
    
    printf("Enter number of elements in the array: ");
    scanf("%d",&n);

    getArray(array,n);
    
    printf("The given array is ");
    dispArray(array,n);
    BubbleSort(array,n);
    printf("\nThe Sorted array is ");
    dispArray(array,n);

}

File: Sorting/In Python #prefer/BubbleSort.py
# Bubble sort in Python
# sinking sort

def bubbleSort(array):
  # loop to access each array element
  for i in range(len(array)):
    # loop to compare array elements
    for j in range(0, len(array) - i - 1):
      # compare two adjacent elements
      # change > to < to sort in descending order
      if array[j] > array[j + 1]:
        # swapping elements if elements
        # are not in the intended order
        temp = array[j]
        array[j] = array[j+1]
        array[j+1] = temp

def getList(array,n):
    for i in range(n):
        print("Enter value {} :".format(i+1))
        val=int(input())
        array.append(val)

n=int(input("\nEnter the number of elements in the list:"))

print("\nEnter the list elements:")
array=[]
getList(array,n)

bubbleSort(array)

print('Sorted Array in Ascending Order:')
print(array)